src/app-layout/classic.tsx

```diff
@@ -303,7 +303,7 @@ const ClassicAppLayout = React.forwardRef(
     const splitPaneAvailableOnTheSide = splitPanelDisplayed && finalSplitPanePosition === 'side';
 
     const sideSplitPanelSize = splitPaneAvailableOnTheSide ? (splitPanelOpen ? splitPanelSize : closedDrawerWidth) : 0;
-    const splitPanelMaxWidth = Math.max(0, resizableSpaceAvailable - effectiveToolsWidth);
+    const sideSplitPanelMaxWidth = Math.max(0, resizableSpaceAvailable - effectiveToolsWidth);
```

Comment:
Renaming just for clarity (this is used only when the split panel is on the side. When it's on the bottom and for the sake of this fix, only `rightOffset` is relevant)